Templates/ServiceBusTopicTrigger-CSharp/function.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "mySbMsg",
+            "accessRights" : "Manage",
+            "type": "serviceBusTrigger",
+            "direction": "in",
+            "topicName": "mytopic",
```

Comment:
Uppercase characters are not allowed in topic names. Hence the topic name in lowercase.